test/e2e/terraform_aks_test.go

```diff
@@ -26,9 +26,17 @@ func TestExamplesStartup(t *testing.T) {
 		aksId, ok := output["test_aks_id"].(string)
 		assert.True(t, ok)
 		assert.Regexp(t, regexp.MustCompile("/subscriptions/.+/resourceGroups/.+/providers/Microsoft.ContainerService/managedClusters/.+"), aksId)
+		assertOutputNotEmpty(t, output, "test_cluster_portal_fqdn")
+		assertOutputNotEmpty(t, output, "test_cluster_private_fqdn")
 	})
 }
 
+func assertOutputNotEmpty(t *testing.T, output test_helper.TerraformOutput, name string) {
+	o, ok := output[name].(string)
+	assert.True(t, ok)
+	assert.NotEqual(t, "", o)
+}
```

Comment:
Is this testing passing when private cluster is not enabled and the value in the tf state is:

```
 "private_fqdn": "",
```
??